yolo.py

```diff
@@ -193,16 +201,17 @@ def evaluate_input(user_input, command):
     print("Modify prompt: ", end = '')
     modded_query = input()
     modded_response = call_open_ai(modded_query)
-    print(modded_response)
     check_for_issue(modded_response)
     check_for_markdown(modded_response)
     modded_user_input = prompt_user_input(modded_response)
     print()
     evaluate_input(modded_user_input, modded_response)
   
   if user_input.upper() == "C":
-    pyperclip.copy(command) 
-    print("Copied command to clipboard.")
+      if os.name == "posix" and missing_posix_display():
+        return
+      pyperclip.copy(command) 
+      print("Copied command to clipboard.")
 
```

Comment:
whoops I forgot the try/except block 🤦🏻‍♂️

Reply:
No worries, I will add in near future. Might also use `return os.environ.get("DISPLAY") is not None` rather then calling `subprocess`, but let me get this change merged. :)